service/src/main/java/cz/muni/fi/pa165/facade/PersonFacadeImpl.java

```diff
@@ -21,7 +21,6 @@
  * @date 14/11/2016
  */
 @Service
-@Transactional
```

Comment:
Tak pri tomto som sa riadil implementaciou ostatnych facadeImpl, tuto to bolo ako jedine a crashovalo to testy.